src/components/Dropdowns/AccessibleDropdown.tsx

```diff
@@ -0,0 +1,233 @@
+import { Button, Dropdown, Tooltip } from "antd";
+import useToken from "antd/es/theme/useToken";
```

Comment:
why not import `import { theme } from 'and'; const { useToken} = theme;`?

Reply:
Fixed, thank you!